index.ts

```diff
@@ -117,6 +118,10 @@ class Api {
         return this.simulator.getState();
     }
 
+    public getAt(address: number): IInstruction {
+        return this.core.getAt(address);
```

Comment:
This should be wrapped in a call to clone to prevent modification of internal state from outside the library